src/themes/default/elements/icon.overrides

```diff
@@ -587,9 +646,42 @@ i.icon.sellsy:before { content: "\f213"; }
 i.icon.shirtsinbulk:before { content: "\f214"; }
 i.icon.simplybuilt:before { content: "\f215"; }
 i.icon.skyatlas:before { content: "\f216"; }
+i.icon.facebook:before { content: "\f230"; }
+i.icon.pinterest:before { content: "\f231"; }
 i.icon.whatsapp:before { content: "\f232"; }
 i.icon.viacoin:before { content: "\f237"; }
 i.icon.medium:before { content: "\f23a"; }
+i.icon.y.combinator:before { content: "\f23b"; }
+i.icon.optinmonster:before { content: "\f23c"; }
+i.icon.opencart:before { content: "\f23d"; }
+i.icon.expeditedssl:before { content: "\f23e"; }
+i.icon.gg:before { content: "\f260"; }
+i.icon.gg.circle:before { content: "\f261"; }
+i.icon.tripadvisor:before { content: "\f262"; }
+i.icon.odnoklassniki:before { content: "\f263"; }
+i.icon.odnoklassniki.square:before { content: "\f264"; }
+i.icon.pocket:before { content: "\f265"; }
+i.icon.wikipedia:before { content: "\f266"; }
+i.icon.safari:before { content: "\f267"; }
+i.icon.chrome:before { content: "\f268"; }
+i.icon.firefox:before { content: "\f269"; }
+i.icon.opera:before { content: "\f26a"; }
+i.icon.internet.explorer:before { content: "\f26b"; }
+i.icon.contao:before { content: "\f26d"; }
+i.icon.five.hundred.pixels:before { content: "\f26e"; }
+i.icon.amazon:before { content: "\f270"; }
+i.icon.houzz:before { content: "\f27c"; }
+i.icon.vimeo:before { content: "\f27d"; }
+i.icon.black.tie:before { content: "\f27e"; }
+i.icon.fonticons:before { content: "\f280"; }
+i.icon.reddit.alien:before { content: "\f281"; }
+i.icon.microsoft.edge:before { content: "\f282"; }
+i.icon.codiepie:before { content: "\f284"; }
+i.icon.modx:before { content: "\f285"; }
+i.icon.fort.awesome:before { content: "\f286"; }
+i.icon.product.hunt:before { content: "\f288"; }
+i.icon.mixcloud:before { content: "\f289"; }
+i.icon.scribd:before { content: "\f28a"; }
```

Comment:
Finally, a lot of new icons for the Networks and Websites category: `microsoft edge` (the browser [?]), `codiepie` (wordpress themes creators), `modx` (web CMS), `fort awesome` (the FontAwesome creators), `product hunt` (the product discovery site), `mixcloud` (the music streaming service), and `scribd` (the digital library)